Utils/github_workflow_scripts/get_xsoar_supported_release_notes.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+"""
+This script is used in the Review Release Notes GitHub workflow.
+It accepts a list of changed files (release notes)
+and returns the release notes of XSOAR-supported Packs.
+"""
+
+from typing import Dict, Any, List
+import json
+from pathlib import Path
+import argparse
+import sys
+import traceback
+import click
+
+
+def is_pack_xsoar_supported(pack_name: str) -> bool:
+    """
+    Checks whether the supplied Pack is from a XSOAR-supported
+
+    Args:
+        - `pack_name` (``str``): The name of the Pack
+
+    Returns:
+        `bool` representing whether Pack is XSOAR-supported or not
+    """
+
+    pack_metadata = Path("Packs", pack_name, "pack_metadata.json")
+
+    if pack_metadata.exists():
+        with pack_metadata.open(encoding="UTF-8") as pm:
+            metadata: Dict[str, Any] = json.load(pm)
+            if metadata.get("support") == "xsoar":
```

Comment:
This should also be done with constants, where do we get the "xsoar" supported constants?